apps/myaccount/src/pages/account-security.tsx

```diff
@@ -135,6 +135,13 @@ const AccountSecurityPage: FunctionComponent<AccountSecurityPagePropsInterface>=
         }
     }, [profileDetails?.profileInfo]);
 
+    /**
+     * Converts the isReadOnlyUserString to a boolean variable
+     * @return {boolean} True/False
+     */
+    const convertToBoolean = (isReadOnlyUserString: any): boolean => {
```

Comment:
Since we are using this method in multiple places, shall we move this to a common file?

Reply:
Fixed by 7e96bf111989b4d81dd8a573bc5ceb710e62d15d.